src/landing-page/index.html

```diff
@@ -6,131 +6,1051 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 
-<body class="hidden">
+<body class="hidden regular-flow">
```

Comment:
this file has a ton of new code. is there a way to load this async if the query param is passed in?